.github/workflows/on-demand-build-docker-image-deploy-preview.yml

```
@@ -5,52 +5,79 @@ on:
       skip-tests: ${{ github.event.client_payload.slash_command.args.named.skip-tests }}
       is-pg-build: ${{ github.event.client_payload.pull_request.base.ref == 'pg' }}
 
   client-build:
+    needs: [resolve-params]
     name: client-build
     if: github.event.client_payload.slash_command.args.named.env != 'release'
     uses: ./.github/workflows/client-build.yml
     secrets: inherit
     with:
-      pr: ${{ github.event.client_payload.pull_request.number }}
+      pr: ${{ needs.resolve-params.outputs.pr_number }}
```

actionlint (1.7.11)

[error] input "pr" is typed as number by reusable workflow "./.github/workflows/client-build.yml". string value cannot be assigned

(expression)